src/interfaces/chain.cpp

```diff
@@ -327,11 +327,13 @@ class ChainImpl : public Chain
     }
     CFeeRate estimateSmartFee(int num_blocks, bool conservative, FeeCalculation* calc) override
     {
-        return ::feeEstimator.estimateSmartFee(num_blocks, calc, conservative);
+        if (!m_node.feeEstimator) return CFeeRate(0);
```

Comment:
I think you can just use default initialization of `CFeeRate`:

```suggestion
        if (!m_node.feeEstimator) return {};
```